src/app/app.css

```diff
@@ -338,6 +338,28 @@ body {
     background-color: #FF9A09
```

Comment:
Try switching to SCSS with variables.

No one writes vanilla CSS, well these guys (https://www.npmjs.com/package/vanilla-css) might disagree, but still learn SCSS.